lib/sidekiq/extensions/generic_proxy.rb

```diff
@@ -24,9 +24,8 @@ def method_missing(name, *args, **kwargs)
         if marshalled.size > SIZE_LIMIT
           ::Sidekiq.logger.warn { "#{@target}.#{name} job argument is #{marshalled.bytesize} bytes, you should refactor it to reduce the size" }
         end
-        @performable.client_push({"class" => @performable,
-                                  "args" => [marshalled],
-                                  "display_class" => "#{@target}.#{name}"}.merge(@opts))
+        item = @performable.serialize({"display_class" => "#{@target}.#{name}"}.merge(@opts), [marshalled])
+        @performable.client_push(item)
```

Comment:
Is this related to "Delayed extensions" that @mperham mentioned? This part of the Sidekiq code looks unfamiliar to me.

Reply:
Yes. For a while now, Sidekiq has included the functionality to take arbitrary model methods and run them asynchronously thru Sidekiq (delaying their execution). When I recreate this branch off of the 7.0 head, this file will be gone I believe and I won't need this change.